verse/stars/reach_at_star.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
```

Comment:
to be overwritten, Nitish has correct version. Combining star sets still not a thing